src/tests/ftest/util/apricot/setup.py

```diff
@@ -0,0 +1,12 @@
+''' apricot setup '''
+from setuptools import setup
+
+setup(name='apricot',
```

Comment:
what is this doing?

Reply:
This is just standard Avocado [subclassing](https://avocado-framework.readthedocs.io/en/latest/SubclassingAvocado.html)